### PR TITLE
Paas 2114

### PR DESCRIPTION
**Pull request by Ronan-Gorain**
JIRA issue: https://jira.jahia.org/browse/PAAS-2114

Short description:

### CODE REVIEW

File: packages/jahia/augmented-search-uninstall.yml [hunk 1]
```diff
@@ -62,3 +63,13 @@ actions:
     - environment.nodegroup.ApplyData[cp]:
         data:
           augsearch: null
+
+  enableDatadogAugSearchCheck:
```
Comment (laurentfufu) [inline packages/jahia/augmented-search-uninstall.yml:67]:
```suggestion
  disableDatadogAugSearchCheck:
```

File: packages/jahia/migrations/migrate-to-v17.yaml [hunk 2]
```diff
@@ -35,6 +35,7 @@ onInstall:
   - updateJelasticRedeployConf        # PAAS-1993
   - setImageMagickMemoryLimit         # PAAS-2024
   - upgradeHaproxy                    # PAAS-1879
+  - enableDatadogAugSearchCheck       # PAAS-2114
```
Comment (laurentfufu) [inline packages/jahia/migrations/migrate-to-v17.yaml:38]:
```suggestion
  - getAugSearchDatadogCheck          # PAAS-2114
```
?

File: packages/jahia/migrations/migrate-to-v17.yaml [hunk 3]
```diff
@@ -149,3 +150,26 @@ actions:
...
+  getAugSearchDatadogCheck:
+    - getPatTokenAndKey
+    - isAugSearchEnabled
+    - cmd[proc,cp]: |-
+        __secret_pat_token="${globals.__secret__pat_token}"
+        check_conf_path="/etc/datadog-agent/conf.d/augmented_search.yaml-disabled"
+        check_conf_path_enabled="/etc/datadog-agent/conf.d/augmented_search.yaml"
+        check_path="/etc/datadog-agent/checks.d/augmented_search.py"
+        if [ ! -f $check_conf_path ] || [ ! -f check_conf_path_enabled ]; then
+          wget -O $check_conf_path ${baseUrl}/v17_assets/augmented_search.yaml
```
Comment (laurentfufu) [inline packages/jahia/migrations/migrate-to-v17.yaml:162]: isn't curl more suitable for error handling ?

File: packages/jahia/migrations/migrate-to-v17.yaml [hunk 4]
```diff
@@ -149,3 +150,26 @@ actions:
...
+          sed -i "s/jahia_root_token_placeholder/$__secret_pat_token/g" $check_conf_path
+          chmod 640 $check_conf_path
+          chown dd-agent: $check_conf_path
+        fi
+        if [ ! -f $check_path ]; then
+          wget -O $check_path ${baseUrl}/v17_assets/augmented_search.py
```
Comment (laurentfufu) [inline packages/jahia/migrations/migrate-to-v17.yaml:168]: isn't curl more suitable for error handling ?

**Review by laurentfufu — CHANGES_REQUESTED** [review]

File: packages/jahia/augmented-search-uninstall.yml (hunk 1 above)
Comment (Ronan-Gorain) [reply packages/jahia/augmented-search-uninstall.yml:67]: :man_facepalming:

File: packages/jahia/migrations/migrate-to-v17.yaml (hunk 2 above)
Comment (Ronan-Gorain) [reply packages/jahia/migrations/migrate-to-v17.yaml:38]: Line lost during conflict with main branche. I set the wrong one when I tried to fix :man_facepalming:

**Pull request merged**